decidim-assemblies/app/commands/decidim/assemblies/admin/update_assembly.rb

```diff
@@ -74,7 +74,28 @@ def attributes
             participatory_scope: form.participatory_scope,
             participatory_structure: form.participatory_structure,
             meta_scope: form.meta_scope,
-            show_statistics: form.show_statistics
+            show_statistics: form.show_statistics,
+            purpose_of_action: form.purpose_of_action,
+            type_of_assembly: form.type_of_assembly,
+            type_of_assembly_other: form.type_of_assembly_other,
+            date_created: form.date_created,
+            created_by: form.created_by,
+            created_by_other: form.created_by_other,
+            duration: form.duration,
+            date_of_inclusion: form.date_of_inclusion,
+            has_closed: form.has_closed,
+            closing_date: form.closing_date,
+            closing_date_reason: form.closing_date_reason,
+            internal_organisation: form.internal_organisation,
+            open_field: form.open_field,
+            public_field: form.public_field,
+            transparent_field: form.transparent_field,
```

Comment:
Wouldn't it be easier to use boolean columns for these three? Also, no need to append `_field`

Reply:
If you think it better to use a Boolean. I change it